src/mjolnir/graphbuilder.cc

```diff
@@ -732,12 +732,11 @@ void BuildTileSet(const std::string& ways_file,
               bike_network = w.bike_network();
             }
 
-            // Add edge info. Mean elevation is set to 1234 as a placeholder, set later if we have it.
             edge_info_offset =
                 graphtile.AddEdgeInfo(edge_pair.second, (*nodes[source]).graph_id,
-                                      (*nodes[target]).graph_id, w.way_id(), 1234, bike_network,
-                                      speed_limit, shape, names, tagged_values, pronunciations, types,
-                                      added, dual_refs);
+                                      (*nodes[target]).graph_id, w.way_id(), kNoElevationData,
```

Comment:
so I changed it from 0 to the no data value, which will return `null` in the geojson response. I traced all usages of `mean_elevation` & co and can't see how this change would affect anything negatively